reportportal/templates/api-deployment.yaml

```diff
@@ -111,6 +111,18 @@ spec:
         - name: RP_BINARYSTORE_MINIO_SECRETKEY
           value: "{{ .Values.minio.secretkey }}"
        {{ end }}
+       {{ if .Values.minio.bucketPrefix }}
+        - name: RP_BINARYSTORE_MINIO_BUCKETPREFIX
+          value: "{{ .Values.minio.bucketPrefix }}"
+       {{ end }}
+       {{ if .Values.minio.defaultBucketName }}
+        - name: RP_BINARYSTORE_MINIO_DEFAULTBUCKETNAME
+          value: "{{ .Values.minio.defaultBucketName }}"
+       {{ end }}
+       {{ if .Values.minio.integrationSaltPath }}
+        - name: RP_INTEGRATION_SALT_PATH
+          value: "{{ default "" .Values.minio.bucketPrefix }}{{ .Values.minio.integrationSaltPath }}"
```

Comment:
No need to add bucketPrefix value here, the prefix will be applied to all bucket name and salt path is one of the buckets.
Have you tested it?

Reply:
We have other internal charts that use the same functionality. I've removed it though.